server/app/acs_enterprise/lambdas/worker_lambda.py

```
@@ -233,17 +233,34 @@ async def process_call(self, message_body: Dict[str, Any], receipt_handle: str,
+                        
+                        if not is_callable:
+                            logger.warning("⚠️ SMS consent not yet obtained - keeping as Ready")
+                            call.status = "Ready"
+                            
+                            # Set blocked_reason in call_metadata for UI visibility
+                            call.call_metadata = {
+                                **(call.call_metadata or {}),
+                                "blocked_reason": "awaiting_sms_consent",
+                                "blocked_at": datetime.now(timezone.utc).isoformat()
+                            }
+                            logger.info(f"🏷️ Set blocked_reason='awaiting_sms_consent' in call_metadata for UI display")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)